sites/public/page_content/locale_overrides/bn.json

```diff
@@ -0,0 +1,117 @@
+{
+  "about.body1": "আমরা জানি যে আপনার চাহিদা পূরণ করে এমন একটি বাড়ি খুঁজে পাওয়া কঠিন এবং হতাশাজনক হতে পারে। ডেট্রয়েট হোম কানেক্ট হল বাড়িতে কল করার জন্য একটি নতুন জায়গা খুঁজে পেতে আপনার সাহায্যকারী হাত।",
+  "about.body2": "ডেট্রয়েট হোম কানেক্ট হল একটি নতুন সিটি অফ ডেট্রয়েট পরিষেবা যা আপনাকে ডেট্রয়েটে আবাসন খোঁজার একটি কেন্দ্রীয় প্রথম ধাপ প্রদান করে যা আপনার সামর্থ্য এবং পরিবারের চাহিদা পূরণ করে। আপনি আপনার পরিবারের আকার, বয়স এবং আয়ের উপর ভিত্তি করে বিকল্পগুলি অন্বেষণ করে ভাড়া ইউনিটের জন্য আপনার যোগ্যতা বুঝতে পারেন। ডেট্রয়েট হোম কানেক্ট হল ডেট্রয়েট শহরের আবাসন ও পুনরুজ্জীবন বিভাগের একটি উদ্যোগ। ওয়েবসাইটের নকশা এবং বৈশিষ্ট্যগুলি এলাকার বাসিন্দা, সম্প্রদায়-ভিত্তিক সংস্থা, সম্পত্তি পরিচালক এবং সম্পত্তির মালিকদের প্রতিক্রিয়া এবং অন্তর্দৃষ্টির উপর ভিত্তি করে।",
+  "about.moreInfoContact": "আরও তথ্যের জন্য, দয়া করে detroithomeconnect@detroitmi.gov-এ সিটি কর্মীদের সাথে যোগাযোগ করুন।",
```

Comment:
Is this correct w the dash attached to the email?

Reply:
This is pulled directly from the Detroit repo. Looks like it's there too